test/docker-compose.yml

```diff
@@ -55,8 +57,6 @@ services:
       POSTGRES_DB: tinkerbell
       POSTGRES_PASSWORD: tinkerbell
       POSTGRES_USER: tinkerbell
-    volumes:
-      - ./db/tinkerbell-init.sql:/docker-entrypoint-initdb.d/tinkerbell-init.sql:ro
```

Comment:
I don't see in this PR what justifies the removal of the `.sql` file (how has this responsibility been shifted elsewhere and to where?)

Reply:
This update follows https://github.com/tinkerbell/sandbox/pull/22 and https://github.com/tinkerbell/sandbox/pull/30. If you look broader on `docker-compose.yml` file changes, you can see that there is new migration container added, which runs before server starts. This container is responsible for seeding the database.

Would you like me to put this in commit message before we merge?

Reply:
Added and also cleaned up git history a bit.